translations/README.md

```diff
@@ -2,7 +2,7 @@
 
 | File            | Language      |
 |-----------------|---------------|
-| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md)                   ￨ [🇬🇷 Ελληνικά](README.greek.md)  |
+| Readme          | [:uk: English](../README.md) ￨ [:fr: Français](README.french.md) ￨ [:jp: 日本語](README.japanese.md) ￨ [🇬🇷 Ελληνικά](README.greek.md) ￨ [:mx: Spanish](README.mx_spanish.md) |
```

Comment:
The Spanish flag is also not showing here.

Reply:
done